orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -1,12 +1,19 @@
-<div class="list-by-status" ng-if="(todos | filter: {skipped_datetime: showSkipped?'!!':'!'}).length > 0">
-  <!-- <h4>{{title}}</h4> -->
-  <div class="todo"
-       ng-repeat="todo in todos | filter: {skipped_datetime: showSkipped?'!!':'!'}">
+<script type="text/ng-template" id="nodes_renderer.html">
+  <div ui-tree-handle class="tree-node tree-node-content">
     <label ng-class="{'text-danger': isInDanger(todo)}">
       <input type="checkbox"
              ng-hide="showSkipped"
              ng-model="todo.completed"
              ng-change="updateTodo(todo)">
+
```

Comment:
What does the heirarchical skipped todo list look like? Can you either get a screenshot or show it a loom video?